### PR TITLE
[zh-cn]: sync translation for File API

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Related issues and pull requests

depends on https://github.com/mdn/content/pull/35756

### CODE REVIEW

File: files/zh-cn/web/api/file/lastmodified/index.md [hunk 1]
```diff
@@ -73,28 +73,28 @@ console.log(fileWithoutDate.lastModified); // 返回当前时间
 
 ## 时间精度降低
 
-为了防止计时攻击和{{glossary("fingerprinting", "指纹识别")}}，`someFile.lastModifiedDate.getTime()` 的精度可能会根据浏览器设置进行舍入。
+为了防止计时攻击和[指纹识别](/zh-CN/docs/Glossary/Fingerprinting)，`someFile.lastModified` 的精度可能会根据浏览器设置进行舍入。在 Firefox 中，`privacy.reduceTimerPrecision` 首选项默认启用，默认为 2ms。您还可以启用 `privacy.resistFingerprinting`，在这种情况下精度将为 100ms 或 `privacy.resistFingerprinting.reduceTimerPrecision.microseconds` 的值，以较大者为准。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/file/lastmodified/index.md:76]:
```suggestion
为了防止计时攻击和[指纹识别](/zh-CN/docs/Glossary/Fingerprinting)，`someFile.lastModified` 的精度可能会根据浏览器设置进行舍入。在 Firefox 中，`privacy.reduceTimerPrecision` 首选项默认启用，默认为 2ms。你还可以启用 `privacy.resistFingerprinting`，在这种情况下精度将为 100ms 或 `privacy.resistFingerprinting.reduceTimerPrecision.microseconds` 的值，以较大者为准。
```

File: files/zh-cn/web/api/file/lastmodifieddate/index.md [hunk 2]
```diff
@@ -26,28 +26,28 @@ for (const file of fileInput.files) {
 
 ## 时间精度降低
 
-为了防止计时攻击和{{glossary("fingerprinting", "指纹识别")}}，`someFile.lastModifiedDate.getTime()` 的精度可能会根据浏览器设置进行舍入。
+为了防止计时攻击和[指纹识别](/zh-CN/docs/Glossary/Fingerprinting)，`someFile.lastModifiedDate` 的精度可能会根据浏览器设置进行舍入。在 Firefox 中，`privacy.reduceTimerPrecision` 首选项默认启用，默认为 2ms。您还可以启用 `privacy.resistFingerprinting`，在这种情况下精度将为 100ms 或 `privacy.resistFingerprinting.reduceTimerPrecision.microseconds` 的值，以较大者为准。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/file/lastmodifieddate/index.md:29]:
```suggestion
为了防止计时攻击和[指纹识别](/zh-CN/docs/Glossary/Fingerprinting)，`someFile.lastModifiedDate` 的精度可能会根据浏览器设置进行舍入。在 Firefox 中，`privacy.reduceTimerPrecision` 首选项默认启用，默认为 2ms。你还可以启用 `privacy.resistFingerprinting`，在这种情况下精度将为 100ms 或 `privacy.resistFingerprinting.reduceTimerPrecision.microseconds` 的值，以较大者为准。
```

File: files/zh-cn/web/api/filelist/index.md [hunk 3]
```diff
@@ -21,8 +21,9 @@ l10n:
 const file = document.getElementById("fileItem").files[0];
 ```
 
-> [!NOTE]
-> 此接口的出现是[对创建不可修改列表的一次尝试](https://stackoverflow.com/questions/74630989/why-use-domstringlist-rather-than-an-array/74641156#74641156)，并且仅是出于避免破坏已经在使用它的代码而继续受到支持。现代 API 使用围绕 ECMAScript 数组类型的类型，因此你可以像 ECMAScript 数组一样对待它们，同时对其使用施加额外的语义（例如使其项目只读）。
+此接口是[尝试创建不可修改的列表](https://stackoverflow.com/questions/74630989/why-use-domstringlist-rather-than-an-array/74641156#74641156)，并且继续支持它只是为了不破坏已经使用它的代码。现代 API 使用基于 JavaScript [数组](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Array)的类型来表示列表结构，从而提供许多数组方法，同时对其使用施加额外的语义（例如使其项目为只读）。
+
+这些历史原因并不意味着您作为开发人员应该避免使用 `FileList`。您不会自己创建 `FileList` 对象，而是从诸如 {{domxref("HTMLInputElement.files")}} 之类的 API 获取它们，并且这些 API 并未弃用。但是，请注意与真实数组的语义差异。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/filelist/index.md:26]:
```suggestion
这些历史原因并不意味着你作为开发人员应该避免使用 `FileList`。您不会自己创建 `FileList` 对象，而是从诸如 {{domxref("HTMLInputElement.files")}} 之类的 API 获取它们，并且这些 API 并未弃用。但是，请注意与真实数组的语义差异。
```

File: files/zh-cn/web/api/filelist/index.md [hunk 4]
```diff
@@ -21,8 +21,9 @@ l10n:
 const file = document.getElementById("fileItem").files[0];
 ```
 
-> [!NOTE]
-> 此接口的出现是[对创建不可修改列表的一次尝试](https://stackoverflow.com/questions/74630989/why-use-domstringlist-rather-than-an-array/74641156#74641156)，并且仅是出于避免破坏已经在使用它的代码而继续受到支持。现代 API 使用围绕 ECMAScript 数组类型的类型，因此你可以像 ECMAScript 数组一样对待它们，同时对其使用施加额外的语义（例如使其项目只读）。
+此接口是[尝试创建不可修改的列表](https://stackoverflow.com/questions/74630989/why-use-domstringlist-rather-than-an-array/74641156#74641156)，并且继续支持它只是为了不破坏已经使用它的代码。现代 API 使用基于 JavaScript [数组](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Array)的类型来表示列表结构，从而提供许多数组方法，同时对其使用施加额外的语义（例如使其项目为只读）。
+
+这些历史原因并不意味着你作为开发人员应该避免使用 `FileList`。您不会自己创建 `FileList` 对象，而是从诸如 {{domxref("HTMLInputElement.files")}} 之类的 API 获取它们，并且这些 API 并未弃用。但是，请注意与真实数组的语义差异。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/filelist/index.md:26]:
```suggestion
这些历史原因并不意味着你作为开发人员应该避免使用 `FileList`。你不会自己创建 `FileList` 对象，而是从诸如 {{domxref("HTMLInputElement.files")}} 之类的 API 获取它们，并且这些 API 并未弃用。但是，请注意与真实数组的语义差异。
```

File: files/zh-cn/web/api/blob/bytes/index.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+---
+title: Blob：bytes() 方法
+short-title: bytes()
+l10n:
+  sourceCommit: de2ef1e9950eebbacdd55f072dfe03014d113bbd
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/blob/bytes/index.md:5]:
```suggestion
  sourceCommit: 5ee9d033cacad3a031562be16be43c55a838cc4f
```

File: files/zh-cn/web/api/filereader/readasarraybuffer/index.md [hunk 6]
```diff
@@ -2,14 +2,15 @@
...
+  sourceCommit: d8f04d843dd81ab8cea1cfc0577ae3c5c9b77d5c
 ---
 
 {{APIRef("File API")}}{{AvailableInWorkers}}
 
 {{domxref("FileReader")}} 接口的 **`readAsArrayBuffer()`** 方法用于开始读取指定 {{domxref("Blob")}} 或 {{domxref("File")}} 的内容。当读取操作完成时，{{domxref("FileReader.readyState","readyState")}} 属性变为 `DONE`，并触发 {{domxref("FileReader/loadend_event", "loadend")}} 事件。此时，{{domxref("FileReader.result","result")}} 属性包含一个表示文件数据的 {{jsxref("ArrayBuffer")}}。
 
-> **备注：** {{domxref("Blob.arrayBuffer()")}} 方法是一种较新的基于 Promise 的 API，用于将文件读取为数组缓冲区。
+> [!NOTE]
+> {{domxref("Blob.arrayBuffer()")}} 方法是一种较新的基于 Promise 的 API，用于将文件读取为数组缓冲区。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/filereader/readasarraybuffer/index.md:13]:
```suggestion
> **备注：** {{domxref("Blob.arrayBuffer()")}} 方法是一种较新的基于 Promise 的 API，用于将文件读取为数组缓冲区。
```

File: files/zh-cn/web/api/filereader/readastext/index.md [hunk 7]
```diff
@@ -2,14 +2,15 @@
...
+  sourceCommit: d8f04d843dd81ab8cea1cfc0577ae3c5c9b77d5c
 ---
 
 {{APIRef("File API")}}
 
 {{domxref("FileReader")}} 接口的 **`readAsText()`** 方法用于读取指定的 {{domxref("Blob")}} 或 {{domxref("File")}} 对象的内容。当读操作完成时，{{domxref("FileReader.readyState","readyState")}} 属性更改为 `DONE`，并触发 {{domxref("FileReader/loadend_event", "loadend")}} 事件，并且 {{domxref("FileReader.result","result")}} 属性包含表示文件内容的文本字符串。
 
-> **备注：** {{domxref("Blob.text()")}} 方法是一种较新的基于 Promise 的 API，用于将文件读取为文本。
+> [!NOTE]
+> {{domxref("Blob.text()")}} 方法是一种较新的基于 Promise 的 API，用于将文件读取为文本。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/filereader/readastext/index.md:13]:
```suggestion
> **备注：** {{domxref("Blob.text()")}} 方法是一种较新的基于 Promise 的 API，用于将文件读取为文本。
```

File: files/zh-cn/web/api/file_api/using_files_from_web_applications/index.md [hunk 8]
```diff
@@ -544,7 +545,7 @@ if (isset($_FILES['myFile'])) {
 
 对象 URL 可以用于图像之外的其他东西！它可以用于显示嵌入的 PDF 文件或任何其他浏览器能显示的资源。
 
-在 Firefox 中，要让 PDF 嵌入式地显示在 iframe 中（而不是作为下载的文件弹出），必须将 `pdfjs.disabled` 设为 `false` {{non-standard_inline()}}.
+在 Firefox 中，要让 PDF 嵌入式地显示在 iframe 中（而不是作为下载的文件弹出），必须将 `pdfjs.disabled` 设为 `false`.
```
Comment (PassionPenguin) [inline files/zh-cn/web/api/file_api/using_files_from_web_applications/index.md:548]:
```suggestion
在 Firefox 中，要让 PDF 嵌入式地显示在 iframe 中（而不是作为下载的文件弹出），必须将 `pdfjs.disabled` 设为 `false`。
```

**Review by PassionPenguin — CHANGES_REQUESTED** [review]
a dot :)

File: files/zh-cn/web/api/filereader/readastext/index.md [hunk 9]
```diff
@@ -2,7 +2,7 @@
 title: FileReader：readAsText() 方法
 slug: Web/API/FileReader/readAsText
 l10n:
-  sourceCommit: e43bfd9b4a6c363a4ba7ef6ffa64c09b38fd111b
+  sourceCommit: d8f04d843dd81ab8cea1cfc0577ae3c5c9b77d5c
 ---
 
 {{APIRef("File API")}}
```
Comment (jasonren0403) [inline files/zh-cn/web/api/filereader/readastext/index.md:8]:
```suggestion
{{APIRef("File API")}}{{AvailableInWorkers}}
```

**Review by jasonren0403 — COMMENTED** [review]
一些建议

**Review by jasonren0403 — APPROVED** [review]
👍

**Pull request merged**